sdk/communication/communication-chat/review/communication-chat.api.md

```diff
@@ -180,79 +215,30 @@ export type ListChatThreadsOptions = RestListChatThreadsOptions;
 // @public
 export type ListMessagesOptions = RestListMessagesOptions;
 
-// @public
-export interface ListPageSettings {
-    continuationToken?: string;
-}
-
 // @public
 export type ListParticipantsOptions = RestListParticipantsOptions;
 
 // @public
 export type ListReadReceiptsOptions = RestListReadReceiptsOptions;
 
 // @public
-export type RemoveParticipantOptions = OperationOptions;
-
-// @public
-export interface RestAddChatParticipantsRequest {
-    participants: RestChatParticipant[];
+export interface MicrosoftTeamsUserIdentifierModel {
+    cloud?: CommunicationCloudEnvironmentModel;
+    isAnonymous?: boolean;
+    userId: string;
 }
 
 // @public
-export interface RestChatMessage {
-    content?: RestChatMessageContent;
-    createdOn: Date;
-    deletedOn?: Date;
-    editedOn?: Date;
-    id: string;
-    senderDisplayName?: string;
-    senderId?: string;
-    sequenceId: string;
-    type: ChatMessageType;
-    version: string;
-}
-
-// @public
-export interface RestChatMessageContent {
-    initiator?: string;
-    message?: string;
-    participants?: RestChatParticipant[];
-    topic?: string;
-}
-
-// @public
-export interface RestChatMessageReadReceipt {
-    chatMessageId: string;
-    readOn: Date;
-    senderId: string;
-}
-
-// @public
-export interface RestChatParticipant {
-    displayName?: string;
-    id: string;
-    shareHistoryTime?: Date;
+export interface PhoneNumberIdentifierModel {
+    value: string;
 }
 
 // @public
-export interface RestChatThread {
-    createdBy: string;
-    createdOn: Date;
-    deletedOn?: Date;
-    id: string;
-    topic: string;
-}
+export type RemoveParticipantOptions = OperationOptions;
 
 // @public
 export interface RestCreateChatThreadOptions extends coreHttp.OperationOptions {
-    repeatabilityRequestID?: string;
-}
-
-// @public
-export interface RestCreateChatThreadRequest {
-    participants: RestChatParticipant[];
-    topic: string;
+    repeatabilityRequestId?: string;
```

Comment:
We had decided to not expose this in the sms library but generate that UUID only internally.

Have we considered doing the same for Chat?

Reply:
In chat it is an optional parameter. If it is not passed, we should generate a UUID (that is what we are doing on the other SDKs). Do we have that fallback as well in js @LuChen-Microsoft  ?

Reply:
Are we generating the UUID in service? Or we need to generate it in SDK

Reply:
The logic should be :
- If the user provides the repeatabilityRequestId, we use it.
- If not, we need to generate a UUID

The service does not generate this. You can take a look at any of the other SDKs as a reference.

Reply:
Just sent a new iteration and fixed this bug. Please check the changes in chatClient.ts